src/components/AttachmentPicker/index.native.js

```diff
@@ -299,15 +303,23 @@ class AttachmentPicker extends Component {
       */
     renderChildren() {
         return this.props.children({
-            openPicker: ({onPicked}) => this.open(onPicked),
+            openPicker: ({onPicked, onClose}) => {
+                this.open(onPicked);
+                if (onClose) {
+                    this.setState({onClose});
+                }
+            },
         });
     }
 
     render() {
         return (
             <>
                 <Popover
-                    onClose={this.close}
+                    onClose={() => {
```

Comment:
@priyeshshah11 we're triggering the focus at `onClose` here so it will face the same issue as the `PopoverMenu` on Android where it fails to open the keyboard. We might want to also use `onModalHide` here instead